Directory.Build.props

```diff
@@ -66,6 +66,8 @@
   <!-- Version properties -->
   <PropertyGroup>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
+	<BuildNumberMajor Condition="'$(BuildNumberMajor)' == ''">00001</BuildNumberMajor>
+    <BuildNumberMinor Condition="'$(BuildNumberMinor)' == ''">0</BuildNumberMinor>
```

Comment:
Just curious. Is there a restriction on the format for BuildNumberMajor that it has to have the four leading zeros?

Reply:
BuildNumberMajor is 5 digit number like 29008 so set default BuildNumberMajor as 00001

---
In reply to: [447871778](https://github.com/dotnet/machinelearning/pull/5268#discussion_r447871778) [](ancestors = 447871778)